### ISSUE TITLE
Superscript tags not rendering as intended

### ISSUE THREAD

**Issue by jtobin** [urbit/urbit.org#1153]
28, 216, and 232 in the following should render as 2^8, 2^16, and 2^32:

![](https://jtobin.s3-ap-southeast-2.amazonaws.com/Screen-Shot-2021-09-22-15-19-49.85.png)

The HTML tags seem correct, so I guess this might be a CSS issue?

```
Urbit IDs are distributed by a sponsorship tree. At the top of the tree
are 2<sup>8</sup> (256) galaxies. Each galaxy issues 2<sup>8</sup>
stars, making a total of 2<sup>16</sup> (65K). Stars then each can issue
2<sup>16</sup> planets, making for 2<sup>32</sup> (~4B). As you might
expect, each planet issues 2<sup>32</sup> moons.
```

**Comment by ohAitch** [comment]
Nonsense, there are only 232 total planets XD

**Comment by g-a-v-i-n** [comment]
nice catch

**Comment by tinnus-napbus** [comment]
Fixed by #1189
